### PR TITLE
Added executive summary and started news page

### PR DESCRIPTION
**Pull request by mhotchkiss84**
What I did:
I created the summary page with an image and flex.
What to expect:
Image, heading tags, and 2 lists. Also used flex box for positioning on the page.

### CODE REVIEW

**Review by swathi862 — CHANGES_REQUESTED** [review]
Everything looks good! However, your execSummary.html file and index.html files are the same. If we merge them into the master copy, we'd just have to delete your index.html since that is already designated as our home page, so no big deal :)

**Review by swathi862 — APPROVED** [review]
Looks great!

File: execSummary.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (swathi862) [inline execSummary.html:7]: Actually, a document title here would be great!

**Review by dconroy91 — COMMENTED** [review]
nice job  guy

**Pull request merged**